layers/blackfire/config.json

```diff
@@ -2,6 +2,7 @@
     "php": [
         "73",
         "74",
-        "80"
+        "80",
+        "81"
```

Comment:
This change is not related, and should be in a different PR

Reply:
Oops, sorry, that shouldn’t be there. Removed.